packages/@ourworldindata/grapher/src/dataTable/DataTable.tsx

```diff
@@ -1079,6 +1205,95 @@ function ClosestTimeNotice({
     )
 }
 
+function Sparkline({
+    width = 75,
+    height = 18,
+    owidRows,
+    minTime,
+    maxTime,
+    highlights = [],
+    dotSize = 3.5,
+    color = "#4C6A9C",
+    strokeStyle = "solid",
+}: {
+    width?: number
+    height?: number
+    owidRows: OwidVariableRow<number>[]
+    minTime: number
+    maxTime: number
+    highlights?: SparklineHighlight[]
+    dotSize?: number
+    color?: string
+    strokeStyle?: "solid" | "dotted"
+}): React.ReactElement | null {
+    if (owidRows.length <= 1) return null
+
+    // add a little padding so the dots don't overflow
+    const bounds = new Bounds(0, 0, width, height).pad(dotSize)
```

Comment:
Should we just `padWidth` here?

That way the dots can overflow in terms of height - _but_ 11px (`18 - 3.5 * 2`) is really not very high at all.
In my mind it's better to have the dots overflow a little bit, with the added benefit of having a slightly more detailed line.

Reply:
Makes sense!